symengine/ntheory.h

```diff
@@ -162,13 +162,19 @@ void nthroot_mod_list(std::vector<RCP<const Integer>> &roots, const RCP<const In
         const RCP<const Integer> &n, const RCP<const Integer> &m);
 //! A solution to x**n == a mod m. Return false if none exists.
 bool nthroot_mod(const Ptr<RCP<const Integer>> &root, const RCP<const Integer> &a,
-        const RCP<const Integer> &n, const RCP<const Integer> &m);
+        const RCP<const Integer> &n, const RCP<const Integer> &mod);
```

Comment:
The function definition in ntheory.cpp used `mod` instead of `m` .Hence I changed `m` to `mod`.